### ISSUE TITLE
Loss function give negative values... bad news ?

### ISSUE THREAD

**Issue by CoteDave** [cpmpercussion/keras-mdn-layer#26]
Hi, 

When training the MDN model, after some times, the custom  "loss=mdn.get_mixture_loss_func(OUTPUT_DIMS,N_MIXES)" give negative values. 

Note: Happens with Standard Scaling on Y. every thing else is quite simple:

![image](https://user-images.githubusercontent.com/49559881/92143349-29a10900-ede3-11ea-8782-a10a106048f7.png)

What that means ? Is it a bug ? Is it still converging ?

thanks!

**Comment by cpmpercussion** [comment]
It's ok for the loss to be negative because of the way that continuous probability density functions work (more discussion in #15 ). 🙃 

Seems weird to me too! But it's generally ok.
